src/constants/contracts.ts

```diff
@@ -446,6 +447,18 @@ export const CONTRACT_TOKEN_ADDRESS = isOnTestnet
       },
     };
 
+interface IContractTokenAddress {
+  id: TokenSymbol;
+  symbol: Uppercase<TokenSymbol>;
+  decimals: number;
+  address: string;
+  asset: string;
+  vasset: string;
+}
+
+export const getContractTokenAddress = (key: TokenSymbol): IContractTokenAddress =>
```

Comment:
The name is a bit misleading to me. We're not just getting the address, we're getting the whole asset.
I think it should just be `getToken` or something like this.